docs/docs/getting-started/choose-provider.md

```diff
@@ -39,6 +39,20 @@ const lucid = await Lucid.new(
 );
 ```
 
+### Maestro
+
+```js
+import { Maestro, Lucid } from "https://deno.land/x/lucid/mod.ts";
+
+const lucid = await Lucid.new(
+  new Maestro(
+    "https://preprod.gomaestro-api.org/",
+    "<project_id>",
```

Comment:
This should be `api-key` not `project_id`. Please update all references below as well.